recipes/xphyle/LICENSE

```diff
@@ -0,0 +1,34 @@
+**MIT License**
```

Comment:
It looks like this license file is already in the pypi src distribution so it can be removed

Reply:
:+1: